rundeckapp/src/test/groovy/rundeckapp/ApplicationTest.groovy

```diff
@@ -68,6 +68,40 @@ class ApplicationTest extends Specification {
 
     }
 
+    def "load rundeck-config.groovy if set in system property RDECK_CONFIG_LOCATION"() {
+        when:
+        File tmpGroovy = File.createTempFile("rundeck-config",".groovy")
+        tmpGroovy << "grails { mail {} } "
+        System.setProperty(RundeckInitConfig.SYS_PROP_RUNDECK_CONFIG_LOCATION,tmpGroovy.absolutePath)
```

Comment:
This test should check if RundeckInitConfig.SYS_PROP_RUNDECK_CONFIG_LOCATION is not set, but there is a rundeck-config.groovy file in the system config directory. If there is a file there, we want to pick it up and load it.

Reply:
https://github.com/danifr/rundeck/blob/28f838350d228f9750aff8048ff366e04e18c65f/rundeckapp/src/test/groovy/rundeckapp/ApplicationTest.groovy#L73